src/components/ConfirmAddLiquidity.vue

```diff
@@ -70,14 +70,17 @@ const namespace = 'addLiquidity'
   }
 })
 export default class ConfirmAddLiquidity extends Mixins(TranslationMixin, DialogMixin) {
-  @Getter('minted', { namespace }) minted!: any
+  @Getter('shareOfPool', { namespace }) shareOfPool!: string
+
```

Comment:
Why do we need space here?

Reply:
Just wanted to refactoring `ConfirmAddLiquidity `and `ConfirmCreatePair ` modals and order their dependency names to equal lists. I'll add A TODO comment for myself, it's ok?